package.json

```diff
@@ -40,14 +40,15 @@
     "@types/node": "^12.0.4",
     "@types/node-fetch": "^2.3.4",
     "concurrently": "^4.1.0",
-    "molstar": "^0.2.2",
+    "molstar": "../molstar",
     "node-fetch": "^2.6.0",
     "tslint": "^5.17.0",
     "typescript": "3.5.1",
     "webpack": "^4.32.2",
     "webpack-cli": "^3.3.2"
   },
   "dependencies": {
-    "argparse": "^1.0.10"
+    "argparse": "^1.0.10",
+    "util.promisify": "^1.0.0"
```

Comment:
please also add this to webpack as an external package `'util.promisify': 'require("util.promisify")',`

Reply:
Done in 7573d26